modules/ztp-configuring-a-static-ip.adoc

```diff
@@ -35,34 +35,33 @@ spec:
      - name: eth0
        type: ethernet
        state: up
-       mac-address: <mac-address> <1>
        ipv4:
          enabled: true
          address:
-           - ip: <ip-address> <2>
-             prefix-length: <public-network-prefix> <3>
+           - ip: <ip-address> <1>
+             prefix-length: <public-network-prefix> <2>
          dhcp: false
    dns-resolver:
      config:
        server:
-         - <dns-resolver> <4>
+         - <dns-resolver> <3>
    routes:
      config:
        - destination: 0.0.0.0/0
-         next-hop-address: <gateway> <5>
+         next-hop-address: <gateway> <4>
          next-hop-interface: eth0
          table-id: 254
  interfaces:
-   - name: "eth0" <6>
-     macAddress: <mac-address> <7>
+   - name: "eth0" <5>
+     macAddress: <mac-address> <6>
 ----
-<1> `mac-address` is the MAC address of the target bare metal machine, that is, the same MAC address used in the `BareMetalHost` resource.
-<2> `ip-address` is the static IP address of the target bare metal machine.
-<3> `public-network-prefix` is the static IP address’s subnet for the target bare metal machine.
-<4> `dns-resolver` is the DNS server for the target bare metal machine.
-<5> `gateway` is the gateway for the target bare metal machine.
-<6> `name` must match the name specified in the `interfaces` section.
-<7> `mac-address` must match the MAC address specified in the `interfaces` section.
+<1> `ip-address` is the static IP address of the target bare metal machine.
```

Comment:
nit: Style guide says to use underscores in variables, eg., `<dns_resolver>`, not `<dns-resolver>`. https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#documentation-guidelines